config/locales/whats_new.yml

```diff
@@ -0,0 +1,67 @@
+en:
+  whats_new:
+    show_banner: false
+    title: What’s new in Manuals Publisher
+    summary: |
+      Summary of updates
+    last_updated: Last updated 3 Jul 2023
+    our_roadmap:
```

Comment:
I imagine it's better to not have a roadmap section given the roadmap we're talking about is measured in weeks.

Maybe you want just a section above the what's new to promote the user research?

Reply:
our UR Rick B was minded to remove user research link